doozerlib/kinit.py

```diff
@@ -0,0 +1,12 @@
+from exectools import cmd_gather
+from exceptions import DoozerFatalError
+
+
+def check_status():
+    rc, out, err = cmd_gather("klist")
+    if rc == 0:
+        if "No credentials cache found" in out:
+            raise DoozerFatalError('Cannot continue because you have no kerberos ticket.')
```

Comment:
Seems a little short for a new file, but I think I get the idea. It didn't really fit in anywhere else. In the future do you think there's anywhere else this could logically live?

Reply:
oh... this is should be removed. Doing now. I realized that it only fails if you have no klist at all, but if you have any it succeeds even if you don't have the right one. It was an experiment.